app/src/main/java/com/willowtree/vocable/presets/PresetsFragment.kt

```diff
@@ -273,20 +228,69 @@ class PresetsFragment : BaseFragment() {
         }
     }
 
+    private fun handleCategories(categories: List<Category>) {
+        with(binding?.categoryView) {
```

Comment:
I think this is one of the places where we can get rid of the `this?.` by using `binding?.categoryView?.apply { }`

Reply:
Good call. I'm wondering if there's a better way to handle binding such that it's a non-nullable type. I feel like it's complicating the code a lot by being nullable. For now I'll update those to use apply.

Reply:
Yeah the docs say that fragment bindings are nullable because the fragment can exist in memory even after its views are destroyed. But I wonder if there's a way around it